src/dfm/cleaning/deduper.py

```diff
@@ -359,43 +424,13 @@ def _deduplicate(
         # If the output file already exists then raise an error if `overwrite`
         # is False and otherwise delete the file
         if output_dir.exists():
```

Comment:
This line is not needed, as it is dealt with in the `save_to_disk` method.

Reply:
It shouldn't always save to disk here. E.g. it should save an empty deduper.

Reply:
Ah right, good point! Can you change the comment above (lines 359-360) to explain this?